src/api/authenticationAPI.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { MOCK } from "./config";
+import AuthenticationAPIMock from "./mock/authenticationAPI";
+
+/* eslint-disable consistent-return */
+export default class AuthenticationAPI {
+  static login(userDetails) {
+    if (MOCK) return AuthenticationAPIMock.login(userDetails);
+
+    return axios
```

Comment:
since you have rebased @Bryan-Cee work it would be better to use the axios instance for consistency.

Reply:
I made the changes am now using axios instance

Reply:
But why is it showing axios???

Reply:
The changes are still local

Reply:
But I guess the aim of changing it to the instance was to make the codebase uniform and also to say that you changed and that the changes are local doesn't pan out as professional.